### PR TITLE
Add references to react-ui-components.

### PR DESCRIPTION
**Pull request by martinpovolny**
Add references to react-ui-components.

### CODE REVIEW

**Comment by martinpovolny** [comment]
ping @mzazrivec , @himdel

File: ui/quadicons.md [hunk 1]
```diff
@@ -43,3 +43,5 @@ end
 ```
 
 It's not recommended to use too much inheritance between decorators, because there might be side effects and we would like to have everything as explicit as possible. Any common behavior that affects all of the quadicons should be moved into a static method in `app/helpers/quadicon_helper.rb`, see `QuadiconHelper.machine_state` as an example.
+
+Old quadicon implementation lives in (obsolote) [ui-components repository](https://github.com/ManageIQ/ui-components). The new one in in `Quadicon` component under the [react-ui-components](https://github.com/ManageIQ/react-ui-components/tree/master/src/quadicon).
```
Comment (mzazrivec) [inline ui/quadicons.md]: obsolote → obsolete

Comment (mzazrivec) [reply ui/quadicons.md]: in in → lives in

File: ui/toolbars.md [hunk 2]
```diff
@@ -77,7 +77,9 @@ Further values have to be passed in as hash. In the example above it is
 Very important key is
   * :`klass`.
 
-Toolbars are rendered into JSON by the ManageIQ and then processed by toolbar component from the [ui-components repository](https://github.com/ManageIQ/ui-components)
+Toolbars are rendered by the ManageIQ using the [`MiqToolbar` component](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/components/miq-toolbar.jsx). That in turns uses the `Toolbar` and other components from the [react-ui-components repository](https://github.com/ManageIQ/react-ui-components/tree/master/src/toolbar).
```
Comment (mzazrivec) [inline ui/toolbars.md]: That in turns → That in turn

**Comment by martinpovolny** [comment]
Thanks for the proof-reading, @mzazrivec. All fixed.

**Comment by mzazrivec** [comment]
Approved. Although I cannot merge, someone else has to do it.

**Comment by himdel** [comment]
Done it :) LGTM too :)

**Pull request merged**